imagetest/test_suites/metadata/setup.go

```diff
@@ -1,17 +1,87 @@
 package metadata
 
 import (
+	"fmt"
+	"math/rand"
+
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest"
 )
 
 // Name is the name of the test package. It must match the directory name.
 var Name = "metadata"
 
+const (
+	shutdownScript = `#!/bin/bash
+
+while [[ 1 ]]; do
+  date +%s >> /shutdown.txt
+  sync
+  sleep 1
+done`
+	shutdownScriptTemplate = `#!/bin/bash
+echo "%s" > %s`
+	shutdownContent    = "The shutdown script worked."
+	shutdownOutputPath = "/shutdown_out.txt"
+	shutdownMaxLength  = 32768 // max shutdown metadata value
+	shutdownScriptURL  = "gs://gcp-guest-cloud-test-artifacts/shutdown-script.txt"
+)
+
 // TestSetup sets up the test workflow.
 func TestSetup(t *imagetest.TestWorkflow) error {
-	_, err := t.CreateTestVM("vm")
+	vm, err := t.CreateTestVM("vm")
+	if err != nil {
+		return err
+	}
+	vm.RunTests("TestTokenFetch|TestMetaDataResponseHeaders|TestGetMetaDataUsingIP")
+
+	vm2, err := t.CreateTestVM("vm2")
+	if err != nil {
+		return err
+	}
+	vm2.SetShutdownScript(fmt.Sprintf(shutdownScriptTemplate, shutdownContent, shutdownOutputPath))
```

Comment:
is there some reason we need to store a template as a constant and then modify it before use, here? it's only used in this one place. the python test had templates for code re-use between startup and shutdown scripts.

Reply:
Right, it's not pretty useful here to use template. Remove